libs/ui/src/pure/Tooltip/index.tsx

```diff
@@ -55,27 +57,48 @@ export function MouseoverTooltipContent({
 }: Omit<TooltipContentProps, 'show'>) {
   const [sticky, setSticky] = useState(false)
   const [show, setShow] = useState(false)
+  const cancelCloseRef = useRef<(() => void) | null>()
```

Comment:
If the types is available in this lib/app without any weird dependencies, I suggest to use the `Command`.
Otherwise ignore the comment.

Reply:
It is :)